adoc/admin-software-installation.adoc

```diff
@@ -188,7 +202,72 @@ helm upgrade --tls <RELEASE_NAME> <CHART> [flags]
 helm del --tls <RELEASE_NAME> [flags]
 ----
 
+[#helm-2to3-migration]
+=== Helm 2 to 3 Migration
+[NOTE]
+====
+The process for migrating an installation from Helm 2 to Helm 3 has been documented and tested by the Helm community. Please reference the following links before proceeding.
+
+- https://v3.helm.sh/docs/topics/v2_v3_migration/
+- https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/
+- https://github.com/helm/helm-2to3
+
+====
+Preconditions:
+
+* A healthy CaaSP 5.0 installation with applications deployed using Helm 2 and Tiller.
+* A system where `skuba` and `helm` version 2 have run from previously.
+** The procedure below requires an available internet connection to install the 2to3 plugin.  If the installation is in an air gapped environment, the system may need to be moved back out of the air gapped environment.
```

Comment:
Is it because the user has to install the helm-2to3 plugin?
```
helm3 plugin install https://github.com/helm/helm-2to3.git
```

Otherwise, the system works in the air-gapped env, right?

Reply:
Correct. The script built in to the 2to3 plugin requires a connection back to github. :(
I modified a version of https://github.com/helm/helm-2to3/blob/master/scripts/install_plugin.sh and created a .tar.gz that could be used to install locally, but that doesn't fit our deployment model. We looked at having an .rpm to install 2to3, but in helm the plugins are installed per-user which didn't fit the .rpm install model.  There are some notes in https://github.com/SUSE/avant-garde/issues/1605

Reply:
Thanks for your explanation ^^